frontend/src/views/Login.vue

```diff
@@ -86,7 +86,8 @@ export default {
         .post(this.$constructUrl('/api/auth/_login'),
           {
             username: this.username,
-            password: this.password
+            password: this.password,
+            rol: this.role
```

Comment:
In fact the role should never be sent during login, the backend will never use it